release-blocking-jobs.md

```diff
@@ -25,12 +25,16 @@ The ideal release-blocking job meets the following criteria:
   - this should be a percentile, not a max
 -->
 - It runs at least every 3 hours
-- It passes 75% of all of its runs in the past week (regardless of underlying
-  commit(s))
 - Is capable of passing 3 times in a row against the same commit
 - Is owned by a SIG that is responsive to addressing failures
   - Represented by a configured testgrid alert email (enforced via presubmit)
-- Fails for no more than 10 runs in a row
+- It passes 75% of all of its runs in the past week (regardless of underlying
+  commit(s)) and fails for no more than 10 runs in a row
+  - In the case of failures, there must be an issue in kubernetes/kubernetes
+    detailing that there is at least one person from the owning SIG working to
+    resolve the issue. Said issue must be acknowledged within a shot period
```

Comment:
Just a small detail: there is a typo on `shot` (should be `short`).

Reply:
Thank you! Fixed